descriptions/Engine.idTech2.md

```diff
@@ -0,0 +1 @@
+**id Tech 2** is a proprietary game engine developed by id Software. It has 2 variants, the Quake engine and the Quake II engine respectively used in [Quake](https://steamdb.info/app/2310/) and in [Quake II](https://steamdb.info/app/2320/)
```

Comment:
We have a separate classification for Quake Engine already.

We should probably rename Quake->idTech2, and rename idTech2 to idTech2.5 and clarify that idTech2 is "the Quake Engine" and idTech2.5 is "the Quake II Engine." I've seen this on various wikis and it seems the most consistent way to go.

Reply:
or we could use a weird name like id Tech 2 Quake Edition and id Tech 2 Quake II Edition 
(this is probably worse)